gems/pending/util/duplicate_blocker/dedup_handler.rb

```diff
@@ -63,8 +68,8 @@ def handle(method, *args)
       key = key_generator.call(method, *args)
       desc = descriptor.call(method, *args)
 
-      entry = update_history(key, desc)
-      tripped?(entry) ? on_blocking_call(desc) : method[*args]
+      entry = update_history(key.hash, desc)
+      tripped?(entry, desc) ? on_blocking_call(desc) : method[*args]
```

Comment:
> Every bit of saving is important.

I kind of disagree.  Savings are only important if they make a significant difference.  If you have to change the code to jump through hoops just to save 0.000001%, it's not worth it.